src/box/call.c

```diff
@@ -160,9 +160,8 @@ box_run_on_call(enum iproto_type type, const char *expr, int expr_len,
 	trigger_run(&box_on_call, &ctx);
```

Comment:
Nit: Can we say 'available outside of its translation unit' (or kinda) instead of 'public'? It confuses me initially, because module API functions are also called 'public'.

> Note, this patch renames access_check_call to access_check_lua_call to
> avoid confusion because this function is now public.

Reply:
Sure. Replaced "public" with "global".